che-core-vfs-impl/src/main/java/org/eclipse/che/vfs/impl/fs/FSMountPoint.java

```diff
@@ -1697,10 +1697,7 @@ public void visit(final VirtualFile virtualFile) {
     private String countHashSum(VirtualFile virtualFile, HashFunction hashFunction) throws ServerException {
         final PathLockFactory.PathLock lock = pathLockFactory.getLock(virtualFile.getVirtualFilePath(), false).acquire(LOCK_FILE_TIMEOUT);
         try {
-            final InputStream stream = virtualFile.getContent().getStream();
```

Comment:
I don't understand how we calculate a checksum if we don't use content of virtual file.